app-engine/java/gradle/resources/META-INF/services/org.jetbrains.plugins.gradle.tooling.ModelBuilderService

```diff
@@ -0,0 +1 @@
+com.google.cloud.tools.intellij.appengine.java.gradle.AppEngineGradleModelBuilderService
```

Comment:
Is this another way to define a service implementation, just plain text lines under `services`?

Reply:
this is not really documented anywhere, but this is the method for getting extra information from the Gradle integration (i.e. building extra Gradle model's). there is some discussion here: https://intellij-support.jetbrains.com/hc/en-us/community/posts/206762805-how-to-use-create-a-gradle-custom-model-